spec/index.bs

```diff
@@ -11116,7 +11116,7 @@ dictionary GPURenderPassDepthStencilAttachment {
         is not {{GPULoadOp/"clear"}}.
 
         The value will be converted to the type of the stencil aspect of |view| by taking the same
-        number of LSBs as the number of bits in the stencil aspect of one texel block of |view|.
+        number of LSBs as the number of bits in the stencil aspect of one [=texel block=] of |view|.
```

Comment:
This is confusing to read since stencil formats kind of fundamentally always have 1x1 blocks. Maybe we could just say `[=texel block|texel=]` and get away with that?

Reply:
Works for me!